app/backend/app.py

```diff
@@ -76,7 +84,8 @@
     "AZURE_AI_TRANSLATION_DOMAIN": "api.cognitive.microsofttranslator.com",
     "BING_SEARCH_ENDPOINT": "https://api.bing.microsoft.com/",
     "BING_SEARCH_KEY": "",
-    "ENABLE_BING_SAFE_SEARCH": "true"   
+    "ENABLE_BING_SAFE_SEARCH": "true" ,
+    "STREAMLIT_HOST_URI":'127.0.0.1'  
```

Comment:
should we remove any legacy streamlit stuff?

Reply:
Done. There should be no mentions of streamlit besides AskData since I haven't completed the transition. However the mentions are all commented out so it will not call streamlit anymore